python/.gitignore

```diff
@@ -45,3 +45,7 @@ pyarrow/_table_api.h
 manylinux1/arrow
 nm_arrow.log
 visible_symbols.log
+
+# don't repeat the text files
```

Comment:
Can we make the comment more explanatory? "Don't repeat" doesn't explain why they would be repeated.